### ISSUE TITLE
Question regarding the dataset layout

### ISSUE THREAD

**Issue by meifish** [joaanna/something_else#13]
Hi!

I tried to train the model, and just a tiny question regarding the structure of the `--root_frames` directory. As instructed in README, I created a folder `frames` and inside the directory, each subfolder is named by the video id, i.e., the integer video id, and contain the break-down frames of the video, i.e., 0001.jpg. Then this directory `./frames/` is used for the `root_frames` argument. 

However, when I started training, it doesn't seem to be able to find the frames. The error is like:
`[WinError 3] The system cannot find the path specified: './toy_dataset/frames/frames/67698'` 

It seems coming from `data_loader_frames.py`  line 115, where a literal `frames/` is hardcoded in the path that causes the double naming. I wonder what is the dataset structure of the root_frames should be like?

Thanks!

**Comment by tjinjin95** [comment]
hi, I have the same problem. Have you solved it? @meifish @joaanna @xiaolonw

**Comment by meifish** [comment]
Line 115 in `data_loader_frames.py`, there is a literal `frames/' in the  path, remove that, and also check if other path has similar issues.

**Comment by hao-zheng-uoa** [comment]
hi @meifish @xiaolonw @joaanna , I am confused how to create the root_frames directory. Should the --root_frames directory be the original something-something v2 video dataset?

**Comment by meifish** [comment]
@HaoShawnZheng :

Not exactly, you need to extract the frames from the original videos first, so each videos will be broken down into N frames. The directory will look like:

| Root project folder
-->| Frames 
--------> | 67698
-----------------> | 0001.jpg
 -----------------> 0002.jpg
 -----------------> 0003.jpg

Your  --root_frame argument should be "/Root_project_folder/frames/". If this doesn't work, try "/Root_project_folder/". In my last attempt around a year ago, the "/frames/" part was harcoded in the data_loader_frames.py. 

Just try run it, and see what you get from the error.
